scripts/create-initial-user.sh

```
@@ -0,0 +1,145 @@
+requireConfig KEYCLOAK_PASSWORD
+
+##############################
+# Keycloak user
+##############################
+
+if ! getKeycloakToken; then
+  echo "ERROR: could not authenticate against Keycloak. Abort."
+  exit 1
+fi
+
+userId=$(curl -s -H "Authorization: Bearer $token" \
```

Shellcheck (0.11.0)

[warning] token is referenced but not assigned.

(SC2154)